api-model/src/main/java/io/enmasse/iot/model/v1/IoTConfigStatus.java

```diff
@@ -26,10 +27,12 @@
 public class IoTConfigStatus {
 
     private boolean initialized = false;
-    private String state;
+    private String phase;
+    private String phaseReason;
```

Comment:
I wonder if this should be an array in case there are multiple reasons (similar to address space status), or do you think there will only be one?